src/components/Calendar/CalendarRoot.tsx

```diff
@@ -223,6 +224,18 @@ class ScheduleCalendar extends PureComponent<ScheduleCalendarProps, ScheduleCale
         canvas.style.color = 'black';
         calendarHeader.style.marginRight = '0px';
 
+        // Set each rbc header and time slot text color to white if the theme is dark
+        const rbc_headers = document.getElementsByClassName('rbc-header');
```

Comment:
I'd like to avoid using any native `document.getElement...` functions because it's hard to tell what exactly they're changing. Can you find the JSX component that needs these styles and apply them there? If that isn't possible, just comment what element this is changing. It can be in the same code comment that resolves my above code review comment.